aptos-move/framework/src/natives/mod.rs

```diff
@@ -32,6 +35,7 @@ pub mod status {
 #[derive(Debug, Clone)]
 pub struct GasParameters {
     pub account: account::GasParameters,
+    pub groups: groups::arkworks_backend::GasParameters,
```

Comment:
Rewrite this as `cryptography::groups` for extra clarity.

Reply:
Fixed.